work.py

```diff
@@ -0,0 +1,85 @@
+print( "WELCOME TO THE PHONEBOOK DIRECTORY") 
+ 
+
+filename = "myphonebook.txt" 
+myfile = open(filename, "a+") 
+myfile.close 
```

Comment:
```suggestion
myfile.close()
```
Это метод, а не атрибут. Методы вызываемы.